crowbar_framework/app/models/node_object.rb

```diff
@@ -170,6 +170,10 @@ def self.all
     self.find nil
   end
 
+  def self.admin_node
+    all.detect(&:admin?)
```

Comment:
We already have it like `all.detect(&:admin?)` within another pull request created by @rsalevsky, that's why I added a comment for using this.